src/result_gen.py

```diff
@@ -0,0 +1,563 @@
+import os
+import sys
+import json
+import shutil
+import random
+import numpy as np
+from bunch import Bunch
```

Comment:
I like bunch. 
Maybe we can use it instead `dotdict` https://github.com/Welthungerhilfe/cgm-ml/blob/9f031bbf2f8396fd568e9adcb0c3eb22a7b91a11/src/common/eval/QA/eval-depthmap-height/src/qa_config.py#L1